### PR TITLE
3.72 release notes content

### PR DESCRIPTION
**Pull request by kmckeedocs**
Note deployment was delayed, and so were release notes for this release. Agent point persons, Mike Moore added for optional review. Amrita not available in github. Gayle, required review.

### CODE REVIEW

File: content/release/ReleaseNotes372.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+## On-premises release date:
+March 31, 2020
+
+### New features and improvements
+
+**Python for Assess**
+This feature is now generally available for all Assess users to instrument web applications written in Python to find custom code vulnerabilities. Assess capabilities include support for Django, Flask, and Pyramid along with route coverage and route-based auto-verification for each framework.
+
+### Important note
+With the 3.7.1 release, which upgraded Tomcat to version 9, users were experiencing a broken AJP (Apache JServ Protocol) connection. This required an AJP secret, or that the secret be disabled. This is configurable through server.properties.
```
Comment (gdvdsn-contrast) [inline content/release/ReleaseNotes372.md:16]: I think the last sentence could be: This is configurable through the _server.properties_ file.

Do you want me to edit the .md to make this change?

File: content/release/ReleaseNotes372.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+## Agent Updates
+
+### Java Agent Summary
+
+**Agent versions released this month:** 3.7.2.14224, 3.7.2.14458
+
+**Language versions supported:** Java 1.6 - 1.8 and 11
+
+#### Bug Fixes:
+* In some cases, when users sanitized attacker-controlled data encoders, Contrast Assess considered the encoder input to also be safe in other contexts. This could have led to false negatives in our data flow analysis. We fixed this so that only the data returned by an encoder (and never the input) would be considered safe in a data flow analysis.
```
Comment (DanFiedler) [inline content/release/ReleaseNotes372.md:28]: The phrasing of the first sentence is really hard to follow. Would the following be equivalent in meaning? 

In some cases, Contrast considered the input to user-specified sanitizers to also be safe in other contexts.

File: content/release/ReleaseNotes372.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
+<!--
+title: "Contrast 3.7.2 - March 2020"
+description: "Contrast 3.7.2 March 2020"
+tags: "3.7.2 March Monthly Release News"
```
Comment (miketmoore) [inline content/release/ReleaseNotes372.md:4]: The front matter configuration looks correct.

File: content/release/ReleaseNotes372.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+-->
+
+## On-premises release date:
+March 31, 2020
```
Comment (DanFiedler) [inline content/release/ReleaseNotes372.md:8]: Does this date need to be changed - did we really release on 3/31?

File: content/release/ReleaseNotes372.md (hunk 2 above)
Comment (gilday) [reply content/release/ReleaseNotes372.md:28]: When you change "users sanitized" to "user-specified sanitizers", you change the semantics of the sentence in such a way that it is no longer accurate. I think it works if we just remove "user-specified"

> In some cases, Contrast considered the input to sanitizers to also be safe in other contexts.

File: content/release/ReleaseNotes372.md (hunk 4 above)
Comment (kmckeedocs) [reply content/release/ReleaseNotes372.md:8]: Thank you Dan.

File: content/release/ReleaseNotes372.md (hunk 2 above)
Comment (kmckeedocs) [reply content/release/ReleaseNotes372.md:28]: Keeping Assess:
In some cases, Contrast Assess considered the input to sanitizers to also be safe in other contexts.

File: content/release/ReleaseNotes372.md (hunk 1 above)
Comment (kmckeedocs) [reply content/release/ReleaseNotes372.md]: This has been completed.

**Pull request merged**